src/plots/gl3d/scene.js

```diff
@@ -88,24 +88,27 @@ function render(scene) {
 
         var tx;
 
-        if(trace.type === 'cone') {
-            var coneTx = [];
+        if(trace.type === 'cone' || trace.type === 'streamtube') {
+            var vectorTx = [];
             if(isHoverinfoAll || hoverinfoParts.indexOf('u') !== -1) {
-                coneTx.push('u: ' + formatter('xaxis', selection.traceCoordinate[3]));
+                vectorTx.push('u: ' + formatter('xaxis', selection.traceCoordinate[3]));
             }
             if(isHoverinfoAll || hoverinfoParts.indexOf('v') !== -1) {
-                coneTx.push('v: ' + formatter('yaxis', selection.traceCoordinate[4]));
+                vectorTx.push('v: ' + formatter('yaxis', selection.traceCoordinate[4]));
             }
             if(isHoverinfoAll || hoverinfoParts.indexOf('w') !== -1) {
-                coneTx.push('w: ' + formatter('zaxis', selection.traceCoordinate[5]));
+                vectorTx.push('w: ' + formatter('zaxis', selection.traceCoordinate[5]));
             }
             if(isHoverinfoAll || hoverinfoParts.indexOf('norm') !== -1) {
-                coneTx.push('norm: ' + selection.traceCoordinate[6].toPrecision(3));
+                vectorTx.push('norm: ' + selection.traceCoordinate[6].toPrecision(3));
+            }
+            if(isHoverinfoAll || hoverinfoParts.indexOf('divergence') !== -1) {
+                vectorTx.push('divergence: ' + selection.traceCoordinate[7].toPrecision(3));
```

Comment:
`cone` with `hoverinfo:'all'`?

Reply:
done in https://github.com/plotly/plotly.js/pull/2658/commits/9a532508dcf4bdff899e647e1c1995f924271500